iam-login-service/src/main/resources/application.yml

```diff
@@ -154,6 +154,11 @@ iam:
     oidc-issuer: ${IAM_REGISTRATION_OIDC_ISSUER:https://example.org}
     saml-entity-id: ${IAM_REGISTRATION_SAML_ENTITY_ID:urn:example}
     add-nickname-as-attribute: ${IAM_ADD_NICKNAME_AS_ATTRIBUTE:false}
+    default-groups:
```

Comment:
These settings must be removed because this application.yml is the default configuration file. This configuration feature must be properly added to IAM documentation.

Reply:
Done